### PR TITLE
[DOCS] Updates to Explore pages 

### PR DESCRIPTION
**Pull request by jmikell821**
Resolves https://github.com/elastic/security-docs/issues/2226

Previews: 
- [Hosts page](https://security-docs_2295.docs-preview.app.elstc.co/guide/en/security/master/hosts-overview.html)
- [Network page ](https://security-docs_2295.docs-preview.app.elstc.co/guide/en/security/master/network-page-overview.html)
- [Users page](https://security-docs_2295.docs-preview.app.elstc.co/guide/en/security/master/users-page.html)

### CODE REVIEW

File: docs/management/hosts/hosts-overview.asciidoc [hunk 1]
```diff
@@ -19,15 +19,18 @@ Beneath the KPI charts are data tables, categorized by individual tabs, which ar
 * *All hosts*: High-level host details.
 * *Uncommon processes*: Uncommon processes running on hosts.
 * *Anomalies*: Anomalies discovered by machine learning jobs.
-* *Events*: All host events.
-* *External alerts*: Alerts received from external monitoring tools.
-* *Sessions*: Linux process events that you can open in Session View, an investigation tool that allows you to examine Linux process data at a hierarchal level. For more information, refer to <<session-view>>. 
+* *Events*: All host events. To view To view <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```
Comment (nastasha-solomon) [inline docs/management/hosts/hosts-overview.asciidoc:22]:
```suggestion
* *Events*: All host events. To find <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```

File: docs/getting-started/users-page.asciidoc [hunk 2]
```diff
@@ -21,10 +21,9 @@ Beneath the KPI charts are data tables, which are useful for viewing and investi
 * *All users*: A chronological list of unique user names, when they were last active, and the associated domains.
 * *Authentications*: A chronological list of user authentication events and associated details, such as the number of successes and failures, and the hostname of the last successful destination.
 * *Anomalies*: Unusual activity discovered by machine learning jobs that contain user data.
-* *Events*: Ingested events that contain the `user.name` field. You can stack by the `event.action`, `event.dataset`, or `event.module` field.
-* *External alerts*: Alerts received from external monitoring tools. You can stack by the `event.module` or `event.category` field.
+* *Events*: Ingested events that contain the `user.name` field. You can stack by the `event.action`, `event.dataset`, or `event.module` field. To view To view <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```
Comment (nastasha-solomon) [inline docs/getting-started/users-page.asciidoc:24]:
```suggestion
* *Events*: Ingested events that contain the `user.name` field. You can stack by the `event.action`, `event.dataset`, or `event.module` field. To find <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```

Comment (nastasha-solomon) [reply docs/getting-started/users-page.asciidoc:24]: I know "find" has a different connotation than "view" so feel free to reject this edit if it's not the message you want to send.

File: docs/getting-started/explore-intro.asciidoc [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 = Explore
 
-The following section includes an overview of the *Hosts*, *Network*, and *Users* pages.
+The following section includes an overview of the *Hosts*, *Network*, *Users*, and *Threat Intelligence* pages.
```
Comment (nastasha-solomon) [inline docs/getting-started/explore-intro.asciidoc:5]:
```suggestion
The following section includes an overview of the *Hosts*, *Network*, and *Users* pages.
```

Comment (nastasha-solomon) [reply docs/getting-started/explore-intro.asciidoc:5]: AFAIK, the TI page isn't be released for 8.4. I noticed it was visible in previous BCs of 8.4 and ESTEC builds but I'm not seeing it in the latest cloud deployment Joe created (which is running BC4 I believe) or my own ESTEC instance (which is running the latest 8.4 snapshot).

**Review by nastasha-solomon — COMMENTED** [review]
Left some comments and edits for your consideration. Looks great overall though!

File: docs/getting-started/explore-intro.asciidoc (hunk 3 above)
Comment (jmikell821) [reply docs/getting-started/explore-intro.asciidoc:5]: Ah, good to know. Thanks!

File: docs/management/hosts/hosts-overview.asciidoc (hunk 1 above)
Comment (jmikell821) [reply docs/management/hosts/hosts-overview.asciidoc:22]: Changing to `display`.

**Review by jamster10 — COMMENTED** [review]
All looks good, but I think the image for IP detail is old. The new view has tabs at the bottom, much like the other entity pages:

image

What you have there is:
image

File: docs/getting-started/explore-intro.asciidoc (hunk 3 above)
Comment (joepeeples) [inline docs/getting-started/explore-intro.asciidoc:5]: I noticed the same thing that @nastasha-solomon commented about the Threat Intelligence page being absent from the latest builds (my latest shared server is BC3). Should it be removed from this line?

File: docs/getting-started/network-page-overview.asciidoc [hunk 4]
```diff
@@ -27,22 +27,34 @@ There are also tabs for viewing and investigating specific types of data:
...
+An IP's detail page shows information for the selected IP address, including links
 to external sites for verifying the IP address's reputation. By default,
 the external sites are https://talosintelligence.com/[TALOS] and
 https://www.virustotal.com/[VIRUSTOTAL]. <<ip-reputation-links>> describes how
 to configure IP reputation links.
 
+To view an IP's detail page, select an IP address from the Source IPs or Destination IPs table.
+
+[role="screenshot"]
+image::images/IP-detail-pg.png[]
```
Comment (joepeeples) [inline docs/getting-started/network-page-overview.asciidoc:46]: Add alt text:
```suggestion
image::images/IP-detail-pg.png[IP details page]
```

File: docs/getting-started/users-page.asciidoc [hunk 5]
```diff
@@ -10,7 +10,7 @@ image::images/users/users-page.png[]
 
 KPI charts show the total number of users and successful and failed user authentications within the time range specified in the date picker. Data in the KPI charts is visualized through linear and bar graphs.
 
-TIP: Hover your cursor inside a KPI chart to access the actions menu (*...*), where you can perform these actions: inspect, open in Lens, and add to a new or existing case.
+TIP: Hover your cursor inside a KPI chart to display the actions menu (*...*), where you can perform these actions: inspect, open in Lens, and add to a new or existing case.
```
Comment (joepeeples) [inline docs/getting-started/users-page.asciidoc:13]: I think the cursor is understood:
```suggestion
TIP: Hover inside a KPI chart to display the actions menu (*...*), where you can perform these actions: inspect, open in Lens, and add to a new or existing case.
```

File: docs/getting-started/users-page.asciidoc (hunk 2 above)
Comment (joepeeples) [inline docs/getting-started/users-page.asciidoc:24]: @nastasha-solomon's suggestion was marked as resolved, but I'm still seeing the repeated phrase "To view To view":
```suggestion
* *Events*: Ingested events that contain the `user.name` field. You can stack by the `event.action`, `event.dataset`, or `event.module` field. To view <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```

File: docs/management/hosts/hosts-overview.asciidoc (hunk 1 above)
Comment (joepeeples) [inline docs/management/hosts/hosts-overview.asciidoc:22]: Ditto "To view To view"
```suggestion
* *Events*: All host events. To view <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```

File: docs/management/hosts/hosts-overview.asciidoc [hunk 6]
```diff
@@ -19,15 +19,18 @@ Beneath the KPI charts are data tables, categorized by individual tabs, which ar
...
+* *Sessions*: Linux process events that you can open in Session View, an investigation tool that allows you to examine Linux process data at a hierarchal level. For more information, refer to <<session-view>>.
```
Comment (joepeeples) [inline docs/management/hosts/hosts-overview.asciidoc:23]: How about linking in the first sentence, for a shorter overall description:
```suggestion
* *Sessions*: Linux process events that you can open in <<session-view,Session View>>, an investigation tool that allows you to examine Linux process data at a hierarchal level.
```

**Review by joepeeples — COMMENTED** [review]
Added a few small edits, otherwise looks good!

**Review by jamster10 — APPROVED** [review]

**Pull request merged**